src/middlewares/routeActivation.ts

```diff
@@ -0,0 +1,97 @@
+import { Context, Next } from 'koa';
```

Comment:
The logic for destinationProcFilter, delivery is same & the logic for BatchFilter, RtFilter is same. We might need to commonise the logic accordingly and proceed. WDYT ?

Reply:
I intentionally kept these in different functions even though its code repetition but it helps in modifying specific logic in case need arises in future.

Reply:
Yep addressed